examples/cur/reportdefinition.yaml

```diff
@@ -0,0 +1,191 @@
+apiVersion: cur.aws.upbound.io/v1beta1
+kind: ReportDefinition
+metadata:
+  annotations:
+    meta.upbound.io/example-id: cur/v1beta1/reportdefinition
+  labels:
+    testing.upbound.io/example-name: example_cur_report_definition
+  name: example-cur-report-definition
+spec:
+  forProvider:
+    additionalArtifacts:
+    - REDSHIFT
+    - QUICKSIGHT
+    additionalSchemaElements:
+    - RESOURCES
+    compression: GZIP
+    format: textORcsv
+    region: us-east-1
+    s3Bucket: test-definition-bucket
```

Comment:
Please use cross-resource referencing here (e.g. `s3BucketSelector`) instead of putting the name of the bucket directly.